crates/lib3h/src/gateway/gateway_transport.rs

```diff
@@ -277,7 +281,7 @@ impl<T: Transport, D: Dht> P2pGateway<T, D> {
                             let peer = PeerData {
                                 peer_address: peer_address.clone(),
                                 peer_uri: transport_id_to_url(id.clone()),
-                                timestamp: 42, // TODO #166
+                                timestamp: peer_timestamp,
```

Comment:
I think this works for now... eventually we'll probably need to separate the timestamps for the spaces from those for the transport layer... but we can design that when we start getting into rrdht